zipkin-lens/src/components/TracePage/TraceSummaryHeader.jsx

```diff
@@ -104,6 +104,51 @@ const TraceSummaryHeader = React.memo(({ traceSummary, rootSpanIndex }) => {
     ? config.logsUrl.replace('{traceId}', traceSummary.traceId)
     : undefined;
 
+  const archiveWriteUrl = (config.archiveWriteUrl && traceSummary)
+    ? config.archiveWriteUrl
+    : undefined;
+
+  const archiveReadUrl = (config.archiveReadUrl && traceSummary)
+    ? config.archiveReadUrl.replace('{traceId}', traceSummary.traceId)
+    : undefined;
+
+  const archiveClick = useCallback(() => {
+    // We don't store the raw json in the browser yet, so we need to make an
+    // HTTP call to retrieve it again.
+    fetch(`${api.TRACE}/${traceSummary.traceId}`)
+      .then((response) => {
+        if (!response.ok) {
+          throw new Error('Failed to fetch trace from backend');
+        }
+        return response.json();
+      })
+      .then((json) => {
+        fetch(archiveWriteUrl, {
+          method: 'POST',
+          headers: {
+            'Content-Type': 'application/json',
+          },
+          body: JSON.stringify(json),
```

Comment:
We once thought about adding a tag, so it is easier to search for traces manually chosen. Do we want to do this?

ex. add a tag to the root span "archived" -> "" would be simplest

cc @openzipkin/core

Reply:
Sounds reasonable to me. I wonder if a user might have an `archived` tag for their own purposes separate from this use case. If so, perhaps we should prefix the tag with `zipkin` or such so it is clear this is a sort of internal tag not part of user data.